src/app.module.ts

```diff
@@ -34,8 +36,20 @@ import { EventEmitterModule } from '@nestjs/event-emitter';
         SilentBlocksModule,
         OperationStateModule,
         BlockProviderModule,
+        ThrottlerModule.forRoot([
+            {
+                ttl: 60000,
+                limit: 10,
```

Comment:
What's TTL?
Does this mean that I can make only 10 requests in 60 seconds?
We should make this configurable.